pkg/controller/seed-controller-manager/initial-application-installation-controller/controller.go

```diff
@@ -149,7 +149,7 @@ func (r *Reconciler) reconcile(ctx context.Context, cluster *kubermaticv1.Cluste
 	}
 
 	if cluster.Spec.CNIPlugin != nil && cni.IsManagedByAppInfra(cluster.Spec.CNIPlugin.Type, cluster.Spec.CNIPlugin.Version) {
-		ciliumApp, err := getCNIApplicationInstallation(ctx, userClusterClient, cluster.Spec.CNIPlugin.Type)
+		ciliumApp, err := util.GetCNIApplicationInstallation(ctx, userClusterClient, cluster.Spec.CNIPlugin.Type)
```

Comment:
nit: atm we only have cilium, but I think we should call this cniApp, as in the func could return something different in the future.